studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -1,12 +1,14 @@
 import { Filters, LogData, LogsEndpointParams, LogsTableName, SQL_FILTER_TEMPLATES } from '.'
 import dayjs, { Dayjs } from 'dayjs'
-import { get } from 'lodash'
+import { get, isEqual } from 'lodash'
 import { StripeSubscription } from 'components/interfaces/Billing'
 import { useMonaco } from '@monaco-editor/react'
 import logConstants from 'shared-data/logConstants'
 import BackwardIterator from 'components/ui/CodeEditor/Providers/BackwardIterator'
-import { uniqBy } from 'lodash'
+import uniqBy from 'lodash/uniqBy'
 import { useEffect } from 'react'
+import utc from 'dayjs/plugin/utc'
+dayjs.extend(utc)
```

Comment:
do we need this? its already extended in `_app`
ideally we load all the required plugins from dayjs in one place

Reply:
pulled it out and tests pass 👍